### PR TITLE
Hour of MakeCode resources

### PR DESCRIPTION
**Pull request by microsoftsam**
Adding initial Hour Of MakeCode resources. 

Two pages: 
- /hourofmakecode where students are expected to go, and through deep linking end up in the different tutorials of each editor
- /hourofmakecode-teacher for teacher resources giving them hardware links, make and code instruction links. 

Material: (5 Tutorials). 
- Rock Paper micro:bit!: https://makecode.microbit.org/tutorials/hour-of-code/rock-paper-scissors
- Magic Wand: https://makecode.adafruit.com/tutorials/magic-wand
- Minecraft: minecraft setup instructions
- Blinky: https://makecode.chibitronics.com/tutorials/getting-started

### CODE REVIEW

File: docs/hourofcode.html [hunk 1]
```diff
@@ -0,0 +1,639 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:og="http://ogp.me/ns#" xmlns:fb="http://www.facebook.com/2008/fbml" lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<title>Microsoft MakeCode - Home Page</title>
```
Comment (pelikhan) [inline docs/hourofcode.html]: Update...

File: docs/hourofcode-teacher.html [hunk 2]
```diff
@@ -0,0 +1,708 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:og="http://ogp.me/ns#" xmlns:fb="http://www.facebook.com/2008/fbml" lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<title>Microsoft MakeCode - Home Page</title>
```
Comment (pelikhan) [inline docs/hourofcode-teacher.html]: New title

**Comment by pelikhan** [comment]
We’re starting to have a lot of duplicates JavaScript Code. Can it be moved into a separate template page?

File: docs/hourofcode-teacher.html (hunk 2 above)
Comment (microsoftsam) [reply docs/hourofcode-teacher.html]: Done

**Comment by microsoftsam** [comment]
I'll clean it up at a future date. This is mostly throw away in a week.

**Comment by pelikhan** [comment]
Hour of code pages can stay up all year long. They keep getting traffic. Besides, we should submit the URL to code.org to be listed.

**Comment by microsoftsam** [comment]
screen shot 2017-11-29 at 5 31 47 pm

File: docs/hourofcode.html (hunk 1 above)
Comment (microsoftsam) [reply docs/hourofcode.html]: Done.

**Pull request merged**